modules/@apostrophecms/admin-bar/ui/apos/apps/AposAdminBar.js

```diff
@@ -10,7 +10,13 @@ export default function() {
           return window.apos;
         }
       },
-      template: '<component :is="`TheAposAdminBar`" :items="apos.adminBar.items" />'
+      render: function (h) {
```

Comment:
`h` is a Vue convention frequently seen in the documentation of the render function, so I'm following that convention. The verbose name of this argument is `createElement`.